src/test/java/io/knowledgeassets/myskills/server/neo4j/IndexManagerTest.java

```diff
@@ -0,0 +1,79 @@
+package io.knowledgeassets.myskills.server.neo4j;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.neo4j.ogm.session.SessionFactory;
+
+import java.io.IOException;
+import java.nio.file.NoSuchFileException;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+@ExtendWith(MockitoExtension.class)
+public class IndexManagerTest {
+
+	@Mock
+	private SessionFactory sessionFactory;
+
+	private IndexManager indexManager;
+
+	@BeforeEach
+	public void setUp() throws IOException {
+		initMocks(this);
```

Comment:
No need to call that. `MockitoExtension` takes care of initializing mocks annotated with `@Mock`.